prow/cmd/branchprotector/request.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package main
 
 import (
+	"log"
+
 	branchprotection "k8s.io/test-infra/prow/config"
```

Comment:
nit: this import should be with the other `test-infra` one in a block

Reply:
Done

Reply:
Done